core/pkg/artifacts/utils.go

```diff
@@ -0,0 +1,52 @@
+package artifacts
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/Khan/genqlient/graphql"
+	"github.com/wandb/wandb/core/internal/gql"
+)
+
+const RegistryProjectPrefix = "wandb-registry-"
+
+func IsArtifactRegistryProject(project string) bool {
+	return strings.HasPrefix(project, RegistryProjectPrefix)
+}
+
+// GetGraphQLInputFields returns the input fields for a given GraphQL input type name into a string array.
```

Comment:
> into a string array

is redundant as it's already implied by the function signature

```suggestion
// GetGraphQLInputFields returns the fields of a GraphQL input type given its name.
```